app/assets/stylesheets/components/_map.scss

```diff
@@ -0,0 +1,5 @@
+.map-container {
```

Comment:
I created the .map-container, removing the inline styling and giving a central location to create map styling.